File: FrontEnd/ClientApp/src/models/playerStatistics.ts
export class PlayerStatistics {
  gamesAll: number = 0;
  minutes: number = 0;
  gols: number = 0;

  gamesHolder: number = 0;
  gamesIn: number = 0; // Subentrato
  gamesOut: number = 0; // Sotituito

  lastGame: Date = null;
  lastGameHolder: Date = null;
}

File: AllenatoreAPI/Models/PlayerStatistics.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AllenatoreAPI.Models
{
    public class PlayerStatistics
    {
        public PlayerStatistics()
        {

        }

        public int GamesAll { get; set; }

        public int Minutes { get; set; }

        public int Gols { get; set; }

        public int GamesHolder { get; set; }

        // Subentrato
        public int GamesIn { get; set; }

        // Sostituito
        public int GamesOut { get; set; }

        public DateTime? LastGame { get; set; }

        public DateTime? LastGameHolder { get; set; }
    }
}
